base.css

```diff
@@ -1,10 +1,89 @@
-@import "intro.css";
-@import "index.css";
-
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-size: 100%;
   font-family: "Roboto", sans-serif;
 }
+
+.gridLayout {
+  position: absolute;
+  min-width: 414px;
+  height: 100vh;
+  background-color: RGB(25, 30, 40);
+  display: flex;
+  flex-direction: column;
+}
+
+.navigationBar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 400px;
+  height: 75px;
+  transition-duration: 5s;
+}
+
+.navigationBarToggle {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  background-color: rgba(200, 200, 200, 0.1);
+  width: 400px;
+  height: 75px;
+
+  transition-duration: 5s;
+}
+.mainArea {
+  width: auto;
+  display: flex;
+  flex-direction: column;
+  background-color: transparent;
+
+  margin-left: 2%;
+}
+
+.menuIcon {
+  float: right;
+  margin: 3%;
```

Comment:
Instead of using percentages for margins, you could use ems (relative to the font size of the element)
Also think if what you need is probably some padding instead of margin

Reply:
HI Namir, iam going to change the entire app to a complete responsive setup.
I will implement ratios instead of static values.

Thanks for the review, i will focus to improve those issues in the future.